From 25c78d7ab57a85e57927bd608b9a7794247072a8 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Sun, 3 Apr 2022 16:08:28 +0300
Subject: [PATCH] GLODROID: HACK: Reduce Bluetooth HCI init timeout to 1s

Sometimes HCI init fails during wakeup which stucks wakeup process for 10s.

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 include/net/bluetooth/hci.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/net/bluetooth/hci.h b/include/net/bluetooth/hci.h
index a5495a3c7091..9992afb8da13 100644
--- a/include/net/bluetooth/hci.h
+++ b/include/net/bluetooth/hci.h
@@ -353,7 +353,7 @@ enum {
 /* HCI timeouts */
 #define HCI_DISCONN_TIMEOUT	msecs_to_jiffies(2000)	/* 2 seconds */
 #define HCI_PAIRING_TIMEOUT	msecs_to_jiffies(60000)	/* 60 seconds */
-#define HCI_INIT_TIMEOUT	msecs_to_jiffies(10000)	/* 10 seconds */
+#define HCI_INIT_TIMEOUT	msecs_to_jiffies(1000)	/* 1 seconds !!! Changed BY GD */
 #define HCI_CMD_TIMEOUT		msecs_to_jiffies(2000)	/* 2 seconds */
 #define HCI_NCMD_TIMEOUT	msecs_to_jiffies(4000)	/* 4 seconds */
 #define HCI_ACL_TX_TIMEOUT	msecs_to_jiffies(45000)	/* 45 seconds */
-- 
2.34.1

